challenge04/solution-herbertmcsnout.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+from scipy.sparse import csr_matrix
```

Comment:
Ah, dredd doesn't have scipy.

Reply:
Ohhh, I'll just implement Dijkstra's alg by hand then

Reply:
Up to you, you can make another PR with the change :]